swagger.yaml

```diff
@@ -4290,13 +7455,78 @@ definitions:
         description: JSON-formatted string containing the usage information by feature name
         x-go-custom-tag: gorm:"type:text"
 
+  infraenv-create-params:
```

Comment:
Do we want to include platform/architecture as part of this PR or in a different pr?